pkg/osquery/tables/dev_table_tooling/table.go

```diff
@@ -0,0 +1,71 @@
+package dev_table_tooling
+
+import (
+	"context"
+	"encoding/base64"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+// allowedCommand encapsulates the possible binary path(s) of a command allowed to execute
+// along with a strict list of arguments.
+type allowedCommand struct {
+	binPaths []string
+	args     []string
+}
+
+type Table struct {
+	logger log.Logger
+}
+
+func TablePlugin(logger log.Logger) *table.Plugin {
+	columns := []table.ColumnDefinition{
+		table.TextColumn("name"),
+		table.TextColumn("args"),
+		table.TextColumn("output"),
+	}
+
+	tableName := "kolide_dev_table_tooling"
+
+	t := &Table{
+		logger: log.With(logger, "table", tableName),
+	}
+
+	return table.NewPlugin(tableName, columns, t.generate)
+}
+
+func (t *Table) generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	var results []map[string]string
+
+	for _, name := range tablehelpers.GetConstraints(queryContext, "name", tablehelpers.WithDefaults("")) {
+		if name == "" {
+			level.Info(t.logger).Log("msg", "Command name must not be blank")
+			continue
+		}
```

Comment:
This is unlikely to trigger -- if name is nil, we won't enter the forloop. But there's no harm in it.